Lesson16/selcan/script.js

```diff
@@ -0,0 +1,304 @@
+    return false;
+  }
+
+}
+console.log(isFirstLetterUppercase("Televiision"));
+
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function ifPositive(number){
+	if(number > 0){
+		return "Positive";
+	}else if(number===0){
+		return "Zero";
+	}else{
+		return "negative";
+	}
+}
+console.log(Exercise5,ifPositive(76));
+
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function isYears(year){
+	if (year % 4 === 0 && year % 100 !== 0 && year % 400 !== 0)
+	{
+        return "It is a leap year."
+	} else {
+		return "It isn't a leap year."
+	}	
+}
+console.log(isYears(2018));
+
+// 7. Write a function that takes in a temperature and checks if it's above or below freezing using if else statement.
+function isTemperatureAbove(degree){
+	if (degree < 0){
+		return "Exercise7;It is below freezing";
+	} else {
+		return "Exercise7; It is above freezing"
+	}
+}
+console.log(isTemperatureAbove(4));
+
+// 8. Write a function that takes in a person's height and weight and checks if they are considered underweight, normal, overweight, or obese using if else statement.
+// Body Mass Index (BMI) is a person’s weight in kilograms (or pounds) divided by the square of height in meters (or feet). A high BMI can indicate high body fatness.
+//Formula: weight (kg) / [height (m)]2
+// Below 18.5	Underweight
+// 18.5 – 24.9	Healthy Weight
+// 25.0 – 29.9	Overweight
+// 30.0 and Above	Obesity
+
+function BMI(weight,height){
+let sonuc = weight / height*height;
+if (sonuc <18.5){
+	return "Exercise8: Underweight";
+} else if (sonuc >=18.5 && sonuc <24.9){
+	return "Exercise8: Healthy Weight";
+} else if (sonuc >=25 && sonuc <29.9){
+	return "Exercise8: Overweight";
+} else if (sonuc >=30){
+	return "Exercise8: Obesity";}
+}
+console.log(BMI(90,1.70));
+
+// 9. Write a function that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+function isChild(age){
+	if (age>0 && age<12){
+		return "Exercise9:Child";
+	} else if(age>13 && age<19){
+	    return "Exercise9:Teenage";
+    } else if(age>20 && age<64){
+	    return "Exercise9:Adult";
+    } else if(age>65){
+	    return "Exercise9:Senior citizen";}
+	}
+
+	console.log(isChild(34));
+
+// 10. Write a function that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+let kisi = "student";
+function canVote(age){
```

Comment:
This function should take 2 parameters:
1. 'age as a number'
2. 'whether they are a student (yes or no)'. 

The first parameter is expected to be number type, the second is usually used as a boolen type in cases like this.
So the function definition will look like this instead:
`function canVote(age, isStudent){`